en/development/testing.rst

```diff
@@ -994,7 +994,13 @@ token mismatches::
 
 It is also important to enable debug in tests that use tokens to prevent the
 SecurityComponent from thinking the debug token is being used in a non-debug
-environment.
+environment. When testing other with other methods like ``requireSecure()`` you
```

Comment:
did you meant "When testing other action with methods like" ?
Because this sentence looks awkward to me.